sql/01_db_init.sql

```diff
@@ -76,18 +77,20 @@ create table prizes
 
 create table points_history
 (
-  id              INTEGER UNIQUE PRIMARY KEY AUTOINCREMENT,
-  child_id        INTEGER NOT NULL,
-  points_change   INTEGER NOT NULL,
-  id_changing_user INTEGER NOT NULL,
-  points_name            TEXT NOT NULL DEFAULT 'points_name',
-  change_timestamp TEXT NOT NULL,
+  id                        INTEGER UNIQUE PRIMARY KEY AUTOINCREMENT,
+  child_id                  INTEGER NOT NULL,
+  points_change             INTEGER NOT NULL,
+  id_changing_user          INTEGER NOT NULL,
+  points_name               TEXT NOT NULL DEFAULT 'points_name',
+  change_timestamp          TEXT NOT NULL,
+  firstname_changing_user   TEXT,
   FOREIGN KEY (child_id) REFERENCES users (id),
-  FOREIGN KEY (id_changing_user) REFERENCES users (id)
+  FOREIGN KEY (id_changing_user) REFERENCES users (id),
+  FOREIGN KEY (firstname_changing_user) REFERENCES users (last_insert_firstname)
```

Comment:
do imienia użytkownika zmieniającego możemy dostać się JOINem używając `id_changing_user`, więc nie potrzebujemy `firstname_changing_user` (co jeśli będziemy potrzebowali nazwisko?)

Reply:
Oja, racja :D

Reply:
zdecydowanie przez sql join. Mogłem napisać w wymaganiach issue :).

Reply:
Nie trzeba było, to moje zaćmienie umysłu :)

Reply:
Już jest w porządku. Dzięki Leszek za pomoc przy zapytaniu :)